src/util/ssa_expr.h

```diff
@@ -16,15 +16,9 @@ Author: Daniel Kroening, kroening@kroening.com
 class ssa_exprt:public symbol_exprt
 {
 public:
-  ssa_exprt()
-  {
-    set(ID_C_SSA_symbol, true);
-  }
-
   /// Constructor
   /// \param expr: Expression to be converted to SSA symbol
-  explicit ssa_exprt(const exprt &expr):
-    symbol_exprt(expr.type())
+  explicit ssa_exprt(const exprt &expr) : symbol_exprt(ID_nil, expr.type())
   {
```

Comment:
This is the first time you've used the nil ID -- before it has been the empty one.
If you are concerned about any time taken to convert "" into an irep_idt, then use irep_idt().

Reply:
Now using `irep_idt()` here.